docs/development.md

```diff
@@ -4,6 +4,20 @@ hide:
 ---
 # Development
 
+## Building Your Development Environment
+
+### Quickstart Guide
+
+If you're already familiar with writing pylint rules, you can get started quickly by following these steps:
+
+```bash
+poetry shell
+poetry install
+invoke build
```

Comment:
```suggestion
poetry install
poetry run invoke build
```

Since the rest of the doc uses `poetry run x` without the expectation of a shell, I would follow the same here and only use this pattern.